dataprofiler/profilers/numerical_column_stats.py

```diff
@@ -929,7 +936,7 @@ def _correct_bias_kurtosis(match_count: int, biased_kurtosis: float) -> float:
         )
         return kurtosis
 
-    def _estimate_mode_from_histogram(self) -> List[float]:
+    def _estimate_mode_from_histogram(self) -> Union[List[float], List[np.float64]]:
```

Comment:
so `mypy` still complains on the return even with this `return` typing? still need to `cast`, @Sanketh7 ?

Reply:
Casting is needed because mypy's type annotation for `.tolist()` gives it a return type of `Any`.